canonical/canonical.com
# Implement country code to contact fields

## Done

- Added country code dropdown to contact fields
- Auto populates Phone number and Country field based on user's timezone
- Big diff is mostly from the timezone json file

## QA

- Go to https://canonical-com-1274.demos.haus/#get-in-touch and https://canonical-com-1274.demos.haus/contact-us#get-in-touch
- Check that Phone number and Country fields are auto-populated based on your timezone
- Input an invalid phone number, see that field shows error message
- Input a valid phone number, see that all works fine

## Issue / Card

Fixes [WD-11496](https://warthogs.atlassian.net/browse/WD-11496)

## Screenshots

[if relevant, include a screenshot]


[WD-11496]: https://warthogs.atlassian.net/browse/WD-11496?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
